java/java.lsp.server/test/unit/src/org/netbeans/modules/java/lsp/server/protocol/ServerTest.java

```diff
@@ -750,7 +752,12 @@ public void publishDiagnostics(PublishDiagnosticsParams params) {
             }
 
             @Override
-            public void showMessage(MessageParams arg0) {
+            public void showMessage(MessageParams params) {
+                if (Server.INDEXING_COMPLETED.equals(params.getMessage())) {
+                    indexingComplete.countDown();
+                } else {
+                    throw new UnsupportedOperationException("Unexpected message.");
```

Comment:
Is it necessary to be this strict ? If some code attempts to e.g. `StatusDisplayer` just to note something, this test will fail.

Reply:
It behaves in the same way as the rest of the tests here.